src/versions.js

```diff
@@ -53,6 +53,11 @@ const versions = {
                 },
             },
             inputs: {
+                datePickerInput: {
+                    devLibraryVersion: '2.0.0', // Is this correct?
```

Comment:
Me too!
I'm hoping @cbrewer1689 has all the deets.

Reply:
@cbrewer1689 Do you have suggested values to plug into here?  Does anything need to be done here prior to merging this guy?

Reply:
Yes, so, the versions file is meant to keep track of gaps. 

- `devLibraryVersion` houses dev's components version value. It can either be 1.x.x or 2.x.x. 1.x.x indicates that design most do not have mockups of this particular component in their library.
    - The greater than (`>`) and less than (`<`) symbols on the `devLibraryVersion` indicates whether there is a gap and  where that gap is. Dev or Design. Greater than indicates the gap is on design. Less than indicates the gap is on us devs.
- `designLibraryVersion` house design's component version value. Pretty simple if the sketch docs say 2.0.1 for Input, then we need to make sure our Input's `designLibraryVersion` says 2.0.1.
- `designLibraryDoc` houses the link to the component in the sketch library. If there is not one, then we can just put "N/A." If there is a given link the UI will render a clickable link on component's doc and the user can then click and go directly to that components design docs.

So, for this instance of datePickerInput, the `devLibraryVersion` should be set to `1.0.0`, because we were saying there aren't any gaps that dev is aware of because design do not have any mockups of the component. For `designLibraryVersion` and `designLibraryDoc` put "N/A."

Reply:
I am not sure if the above helps or not, but i did update the versions.js commented notes in `dev` in hopes that it is helpful.

Reply:
Very helpful!  Thanks!

Reply:
Updated in 559da21 to the following:
```JavaScript
datePickerInput: {
    devLibraryVersion: '1.0.0',
    designLibraryVersion: 'N/A',
    designLibraryDoc: 'N/A',
},
```